ait/core/table.py

```diff
@@ -468,7 +467,7 @@ def dirty(self):
     def load(self):
         if self.fswtabdict is None:
             if self.dirty():
-                self.fswtabdict = createFSWTabDict(self.filename)
+                self.fswtabdict = FSWTabDict(self.filename)
```

Comment:
Sorry, my comment from last review was incorrect. The `createFSWxxx` method gets created in run-time during the extension process (i.e. `util.__init_extensions__`). So it is okay to call `createFSWxxx` method here. Please revert back to before this commit.